cmd/kpod/export.go

```diff
@@ -0,0 +1,103 @@
+package main
+
+import (
+	"io"
+	"os"
+
+	"fmt"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/containers/storage"
+	"github.com/containers/storage/pkg/archive"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+)
+
+type exportOptions struct {
+	output    string
+	container string
+}
+
+var (
+	exportFlags = []cli.Flag{
+		cli.StringFlag{
+			Name:  "output, o",
+			Usage: "Write to a file, default is STDOUT",
+			Value: "/dev/stdout",
+		},
+	}
+	exportDescription = "Exports container's filesystem contents as a tar archive" +
+		" and saves it on the local machine."
+	exportCommand = cli.Command{
+		Name:        "export",
+		Usage:       "Export container's filesystem contents as a tar archive",
+		Description: exportDescription,
+		Flags:       exportFlags,
+		Action:      exportCmd,
+		ArgsUsage:   "CONTAINER",
+	}
+)
+
+// exportCmd saves a container to a tarball on disk
+func exportCmd(c *cli.Context) error {
+	args := c.Args()
+	if len(args) == 0 {
```

Comment:
You probably figured this out, but just in case.  This error is displayed for the case when a containerID is not part of the command line and also if you used --output and don't give a filename.  When it fires for the lack of the containerID it's a valid error message.  Not so much for the --output case.  I've not spent time looking, but there's gotta be a way to determine if the current param being parsed is the --output and then and only then show the error you're suggesting.  Unfortunately it's not just a fix the error message fix.